sympy/holonomic/holonomic.py

```diff
@@ -1229,6 +1160,200 @@ def _find_conditions(simp, x, x0, order, evalf=False):
     return HolonomicFunction(sol, x).composition(z)
 
 
+x_1 = Dummy('x_1')
+_lookup_table = None
+from sympy.integrals.meijerint import _mytype
+
+
+def from_sympy(func, x=None, initcond=True):
```

Comment:
These names will need to change, or else become methods of a class, if they are ever going to be added to a public namespace.